### PR TITLE
[11.0][MIG] Migration project

### PR DESCRIPTION
**Pull request by yuriqp**
Description of the issue/feature this PR addresses:
Migration of project module

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/project/migrations/11.0.1.1/pre-migration.py
Comment (pedrobaeza) [inline addons/project/migrations/11.0.1.1/pre-migration.py]: calendar?

Comment (pedrobaeza) [reply addons/project/migrations/11.0.1.1/pre-migration.py]: Isn't better to create an openupgradelib function for handling this?

**Review by pedrobaeza — DISMISSED** [review]

File: addons/project/migrations/11.0.1.1/pre-migration.py
Comment (yuriqp) [reply addons/project/migrations/11.0.1.1/pre-migration.py]: I saw that @rruebner already created a PR with this function in openupgradelib (https://github.com/OCA/openupgradelib/pull/87). What should I do, wait until it is merged?

Comment (pedrobaeza) [reply addons/project/migrations/11.0.1.1/pre-migration.py]: Yes, let's merge it and then call it from here.

**Comment by pedrobaeza** [comment]
Still waiting to fix and merge PR in openupgradelib for translation cleanup.

**Comment by pedrobaeza** [comment]
@yuriqp please use the recently merged `delete_record_translations` method in openupgradelib.

**Comment by pedrobaeza** [comment]
Travis is still red

**Comment by yuriqp** [comment]
@pedrobaeza Travis is now green :), can this go further?

**Comment by pedrobaeza** [comment]
I'm not seeing the conversion of all project issues into project tasks. My strategy would be:

* Look for projects with issues.
* Duplicate that projects, with a suffix like " - Issues".
* Create one task per issue associated to the duplicated project.

**Comment by hbrunn** [comment]
...and only create tasks for issues that don't already have a task I'd say

**Comment by yuriqp** [comment]
but wouldn't we be losing some information if we get rid of issues related with tasks? For example, let say I have an issue related with a task, and added some information in the description field of the issue.

**Comment by yuriqp** [comment]
Travis is failing, but looks like it is not related with the changes in the branch.

**Comment by yuriqp** [comment]
Any news on why is travis failing?

**Comment by StefanRijnhart** [comment]
@pedrobaeza thanks for picking this one up. Will you migrate issue messages and attachments to the new tasks as well?

**Comment by pedrobaeza** [comment]
Good catch! I haven't thought about that, but yes, I can do it. For now, I was checking that everything was more or less in line. Adding both things in next push.

**Comment by pedrobaeza** [comment]
@StefanRijnhart This is ready. Maybe we can inject some values for testing followers, but I'm a bit exhaust with this module.

**Review by StefanRijnhart — APPROVED** [review]
Thanks!

**Pull request merged**
